curriculum/challenges/english/25-front-end-development/quiz-relational-database/66f1af82732957c895f0b21a.md

```diff
@@ -17,439 +17,441 @@ Answer all of the questions below correctly to pass the quiz.
 
 ---
 
-Placeholder distractor 3
+ADD DATA
 
 #### --answer--
 
-Placeholder answer
+INSERT INTO
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How do you rename a column in SQL?
 
 #### --distractors--
 
-Placeholder distractor 1
+`MODIFY COLUMN`
 
 ---
 
-Placeholder distractor 2
+`CHANGE COLUMN`
 
 ---
 
-Placeholder distractor 3
+`ALTER COLUMN`
 
 #### --answer--
 
-Placeholder answer
+`RENAME COLUMN`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which query will delete all records from the employees table?
 
 #### --distractors--
 
-Placeholder distractor 1
+REMOVE ALL
 
 ---
 
-Placeholder distractor 2
+TRUNCATE employees
 
 ---
 
-Placeholder distractor 3
+DROP RECORDS
 
 #### --answer--
 
-Placeholder answer
+DELETE FROM employees
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which SQL constraint specifically ensures that a column can contain only unique values, allowing nulls?
 
 #### --distractors--
 
-Placeholder distractor 1
+`NOT NULL`
 
 ---
 
-Placeholder distractor 2
+`FOREIGN KEY`
 
 ---
 
-Placeholder distractor 3
+`PRIMARY KEY`
 
 #### --answer--
 
-Placeholder answer
+`UNIQUE`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+In a relational database, what is an index used for?
 
 #### --distractors--
 
-Placeholder distractor 1
+To store additional metadata about tables
 
 ---
 
-Placeholder distractor 2
+To ensure unique values in a column
 
 ---
 
-Placeholder distractor 3
+To enforce data integrity
 
 #### --answer--
 
-Placeholder answer
+To speed up data retrieval
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does a one-to-one (1-1) relationship represent in a relational database?
 
 #### --distractors--
 
-Placeholder distractor 1
+Each record in Table A corresponds to one or more records in Table B
 
 ---
 
-Placeholder distractor 2
+Each record in Table A corresponds to no records in Table B
 
 ---
 
-Placeholder distractor 3
+Each record in Table A corresponds to multiple records in Table B and vice versa
+
 
 #### --answer--
 
-Placeholder answer
+ Each record in Table A corresponds to exactly one record in Table B
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What command will list all tables in the current PostgreSQL database?
 
 #### --distractors--
 
-Placeholder distractor 1
+`\l`
 
 ---
 
-Placeholder distractor 2
+SHOW TABLES
 
 ---
 
-Placeholder distractor 3
+`\t`
 
 #### --answer--
 
-Placeholder answer
+`\d`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a composite key in a relational database?
 
 #### --distractors--
 
-Placeholder distractor 1
+A key that is derived from another key
 
 ---
 
-Placeholder distractor 2
+A key that changes frequently based on data updates
 
 ---
 
-Placeholder distractor 3
+ A key used exclusively for foreign key relationships
 
 #### --answer--
 
-Placeholder answer
+A key that is composed of multiple columns to uniquely identify a record
+
```

Comment:
```suggestion
A key that is composed of multiple columns to uniquely identify a record
```

Reply:
Please review now!

Reply:
you have also not removed this extra empty line @BethiPooja

Reply:
Please check now!